connector/connect/common/src/main/scala/org/apache/spark/sql/connect/common/DataTypeProtoConverter.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.spark.util.Utils
 /**
  * Helper class for conversions between [[DataType]] and [[proto.DataType]].
  */
-object DataTypeProtoConverter {
+private[sql] object DataTypeProtoConverter {
```

Comment:
@grundprinzip Should the converters in common marked as public? Also see `LiteralValueProtoConverter` and `StorageLevelProtoConverter`

Can we modify them to be package protected?